Src/BootCamp.Chapter1/Hangman.cs

```diff
@@ -23,18 +21,103 @@ public static class Hangman
         /// <returns>True if game was won or false if game was lost.</returns>
         public static bool Play(int lives, string wordsFile, int difficulty)
         {
+            if (difficulty < 3 || lives == 0)
+            {
+                throw new Exception();
+            }
+
             var wordToGuess = WordsBank.PickRandomWord(@$"Words/{wordsFile}", difficulty);
-            // ToDo: finish the game!
-            var playersGuess = GuessCharacter();
+            var anymousWord = ConvertWordSoUserCannotReadit(wordToGuess);
+
+            Console.WriteLine($"Word to guess: { anymousWord} {Environment.NewLine}");
+            var alreadyUsedChars = "";
+
+            do
```

Comment:
PLease split do..while part to a function.

Reply:
I can do that but then I get a function with 4 arguments :  `playGame( lives, wordToGuess, anymousWord, alreadyUsedChars); ` 

and according to your remarks  it is better to have to function with a max of 2 arguments but here I see no way to make that work. 
so please some hints.